pkg/webhooks/regularuser/regularuser.go

```diff
@@ -168,18 +171,26 @@ func (s *RegularuserWebhook) authorized(request admissionctl.Request) admissionc
 		ret.UID = request.AdmissionRequest.UID
 		return ret
 	}
+	if utils.SliceContains("dedicated-admins", request.UserInfo.Groups) &&
```

Comment:
Does this need to _also_ accounts for `cluster-admins` users?

Reply:
If that's in the spec, yes. Let's chat Tuesday.